### PR TITLE
Fix new graphics on corporations

### PR DESCRIPTION
**Pull request by vsrisuknimit**
- Update the corporations with new graphics.

Some examples:

Old
image

Updated
image
image
image

### CODE REVIEW

File: src/HTML_data.ts [hunk 1]
```diff
@@ -5139,7 +5139,7 @@ export const HTML_DATA: Map<string, string> =
                       box-shadow: 6px 6px 6px grey;
                       border-radius:2px;">
                   helion</div><br>
-      <div class="production-box production-box-size3" style="margin-left:20px;margin-top:-50px;">
+      <div class="production-box production-box-size3" style="margin-left:20px;margin-top:-10px;">
```
Comment (alrusdi) [inline src/HTML_data.ts:5142]: Please move inline styles to cards.less

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/HTML_data.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/HTML_data.ts:5142]: @alrusdi There are inline styles everywhere in corporation cards. Do you want me to move just this one box specifically or everywhere?

Full Helion code for example:
https://github.com/bafolts/terraforming-mars/blob/6ec98ef60abb6d5988e1d20cf80b00cf4a8a2901/src/HTML_data.ts#L5119-L5150

Comment (alrusdi) [reply src/HTML_data.ts:5142]: Not all, but if you take some card to fix please do full cleanup of that card. 
In this case move all inline styles of Helion corp to cards.less
If each of us will do it - someday we will finish that cleanup

Comment (vsrisuknimit) [reply src/HTML_data.ts]: @alrusdi I moved all inline style for Helion to card.less. I fixed many other corps too in terms of graphics but I will deal with their styles later if that is okay.

**Review by alrusdi — APPROVED** [review]

**Pull request merged**